ravenpy/models/base.py

```diff
@@ -267,7 +277,7 @@ def setup_model_run(self, ts):
         # Create symbolic link to input files
         for fn in ts:
             if not (self.model_path / Path(fn).name).exists():
-                os.symlink(str(fn), str(self.model_path / Path(fn).name))
+                os.symlink(os.path.realpath(fn), str(self.model_path / Path(fn).name))
```

Comment:
Did you want this to be a strict check for a real path? ie: https://docs.python.org/3/library/os.path.html#os.path.realpath

Reply:
Not sure I understand your question. The problem was that the "source" part of the symlink was not good (in the case that the file is coming from the notebook env) so I replaced it (the source) with the full absolute path. The "destination" stays the same.

Reply:
Ah, I understand now. I also had no idea that `realpath` expanded symlinks. Very useful. (https://stackoverflow.com/a/40311142/7322852)